include/circt/Support/FVInt.h

```diff
@@ -646,8 +646,8 @@ inline FVInt operator-(uint64_t a, const FVInt &b) {
 
 inline FVInt operator-(const APInt &a, const FVInt &b) { return FVInt(a) - b; }
 
-inline bool operator==(uint64_t a, const FVInt &b) { return b == a; }
-inline bool operator!=(uint64_t a, const FVInt &b) { return b != a; }
+inline bool operator==(uint64_t a, const FVInt &b) { return b.operator==(a); }
+inline bool operator!=(uint64_t a, const FVInt &b) { return b.operator!=(a); }
```

Comment:
Any tips on how to best deal with https://timsong-cpp.github.io/cppwp/n4868/over.match#oper-3.4.4 in a codebase that still needs to support older standards?